src/NUnitFramework/framework/Constraints/SomeItemsConstraint.cs

```diff
@@ -56,11 +56,11 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         /// <summary>
         /// Flag the constraint to use the supplied <see cref="Func{TCollectionType, TMemberType, Boolean}"/> object.
         /// </summary>
-        /// <typeparam name="TCollectionType">The type of the elements in the collection.</typeparam>
-        /// <typeparam name="TMemberType">The type of the member.</typeparam>
+        /// <typeparam name="TActualCollectionElement">The type of the elements in the collection.</typeparam>
+        /// <typeparam name="TExpectedElement">The type of the expected element.</typeparam>
```

Comment:
I think I prefer `TExpected` as it must match the type specified in your constraint and is not necessary the same as any element type.

Reply:
Right, good point. I adjusted the XML comment as well.